CHANGES.txt

```diff
@@ -4,6 +4,8 @@
  * Add flag to ignore unreplicated keyspaces during repair (CASSANDRA-15160)
 
 3.0.22:
+ * Check SSTables for latest version before dropping compact storage (CASSANDRA-16063)
+ * Allow skipping commit log replay does not fail on descriptor errors (CASSANDRA-16063)
```

Comment:
I think we don't have any changes for this in this PR.

Reply:
True, leftover from the previous patch version. Removed

Reply:
I think the entry about skipping commit log replay is still there. It seems that something in the rebase is including unrelated commits in the PR.